features/schema_metadata.feature

```diff
@@ -0,0 +1,296 @@
+      <?php
+      $cluster  = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session  = $cluster->connect("simplex");
+      
+      $schema   = $session->schema();
+      $keyspace = $schema->keyspace("simplex");
+
+      echo "Name: " . $keyspace->name() . "\n";
+      echo "Replication Class: " . $keyspace->replicationClassName() . "\n";
+      foreach ($keyspace->replicationOptions() as $key => $value) {
+        echo "  " . $key . ":" . $value . "\n";
+      }
+      echo "Has Durable Writes: " . ($keyspace->hasDurableWrites() ? "True" : "False" ). "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Name: simplex
+      Replication Class: org.apache.cassandra.locator.SimpleStrategy
+        replication_factor:1
+      Has Durable Writes: False
+      """
+
+    Scenario: Table metadata will exist for all major Cassandra versions
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Name: " . $table->name() . "\n";
+      echo "Bloom Filter: " . $table->bloomFilterFPChance() . "\n";
+      $patterns = array();
+      $patterns[0] = '/{/';
+      $patterns[1] = '/"/';
+      $patterns[2] = '/:/';
+      $patterns[3] = '/keys/';
+      $table_caching = explode(",", $table->caching());
+      echo "Caching: " . preg_replace($patterns, "", $table_caching[0]) . "\n";
+      echo "Comment: " . $table->comment() . "\n";
+      echo "Compaction Class: " . $table->compactionStrategyClassName() . "\n";
+      foreach ($table->compactionStrategyOptions() as $key => $value) {
+        echo "  " . $key . ":" . $value . "\n";
+      }
+      foreach ($table->compressionParameters() as $key => $value) {
+        echo "  " . $key . ":" . $value . "\n";
+      }
+      echo "DC/Local Read Repair Chance: " . $table->localReadRepairChance() . "\n";
+      echo "Garbage Collection Grace Seconds: " . $table->gcGraceSeconds() . "\n";
+      echo "Read Repair Chance: " . $table->readRepairChance() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Name: values
+      Bloom Filter: 0.5
+      Caching: ALL
+      Comment: Schema Metadata Feature
+      Compaction Class: org.apache.cassandra.db.compaction.LeveledCompactionStrategy
+        sstable_size_in_mb:37
+        sstable_compression:org.apache.cassandra.io.compress.DeflateCompressor
+      DC/Local Read Repair Chance: 0.25
+      Garbage Collection Grace Seconds: 3600
+      Read Repair Chance: 0.75
+      """
+
+    @cassandra-version-less-2.1
+    Scenario: Additional table metadata will only exist for Cassandra versions
```

Comment:
Scenario name suggestion - Getting table metadata for io cache and replicate on write